code/modules/mob/living/simple_animal/hostile/monster.dm

```diff
@@ -48,4 +59,50 @@
 /obj/item/projectile/energy/neurotox
 	damage = 10
 	damage_type = TOX
-	icon_state = "toxin"
\ No newline at end of file
+	icon_state = "toxin"
+
+/mob/living/simple_animal/hostile/monster/cyber_horror
+	name = "cyber horror"
+	desc = "What was once a man, twisted and warped by machine."
+	icon_state = "cyber_horror"
+	icon_dead = "cyber_horror_dead"
+	icon_gib = "cyber_horror_dead"
+	speak = list("H@!#$$P M@!$#", "GHAA!@@#", "KR@!!N", "K!@@##L!@@ %!@#E", "G@#!$ H@!#%, H!@%%@ @!E")
+	speak_emote = list("emits", "groans")
+	speak_chance = 30
+	turns_per_move = 5
+	see_in_dark = 6
+	maxHealth = 70
+	health = 70
+	melee_damage_lower = 5
+	melee_damage_upper = 15
+	attacktext = "flails around and hits"
+	move_to_delay = 5
+	can_butcher = 0
+	attack_sound = 'sound/weapons/hivehand_empty.ogg'
+
+/mob/living/simple_animal/hostile/monster/cyber_horror/Life(var/mob/living/simple_animal/hostile/monster/cyber_horror/M)
+	..()
+
+	if(prob(90) && health<maxHealth)
+		health=health+2                                                                        //Created by misuse of medical nanobots, so it heals
+		if(prob(25))
```

Comment:
This is still too high, life() occurs every 2 seconds so this occurs every 8. Most people consider a message that occurs more often than 10 or 20 seconds to be spam.